File: app/src/main/java/com/BlizzardArmory/starcraft/Profile/CategoryPointProgress.java
package com.BlizzardArmory.starcraft.Profile;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class CategoryPointProgress {

    @SerializedName("categoryId")
    @Expose
    private String categoryId;
    @SerializedName("pointsEarned")
    @Expose
    private int pointsEarned;

    public String getCategoryId() {
        return categoryId;
    }

    public void setCategoryId(String categoryId) {
        this.categoryId = categoryId;
    }

    public int getPointsEarned() {
        return pointsEarned;
    }

    public void setPointsEarned(int pointsEarned) {
        this.pointsEarned = pointsEarned;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/starcraft/profile/CategoryPointProgress.kt
package com.BlizzardArmory.model.starcraft.profile

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName


/**
 * The type Category point progress.
 */
@Keep
data class CategoryPointProgress(

    @SerializedName("categoryId")
    var categoryId: String,

    @SerializedName("pointsEarned")
    var pointsEarned: Int

)